signac/contrib/project.py

```diff
@@ -732,63 +729,49 @@ def _find_job_ids(self, filter=None, doc_filter=None):
         ValueError
             If the filters are invalid.
 
-        Notes
```

Comment:
I believe this note was added because of some issue in aggregation where passing some other argument was causing severe performance degradation, but since the use case would have to be "not filtering" I'm not sure what that other arg was. Do you remember? I just want to make sure that this is not still relevant.

**EDIT**: See my next comment below, I think this is specific to cases where `JobsCursor` construction occurs but please verify.

Reply:
See reply below -- I think this can be resolved. This was changed in #514 and there is no longer a performance difference between `{}` and `None`.